parts/questions/02-how-data.html.j2

```diff
@@ -265,22 +261,6 @@
         {%- endif -%}
       {%- endif -%}
       </p>
-      {# Graph Database #}
```

Comment:
This was changed probably together with phrasing of this question: `4539a575-6bdd-4077-a2fd-71b8121e2e06` (i.e. it is not only "relational dababase" in the option above but all kinds of databases). Similarly, the option about "object store" is now covering also "document store".